assets/sass/molecules/_card.scss

```diff
@@ -231,6 +233,7 @@ a.card-img-top {
 
 .card-white { @include card-bg-variant($white); }
```

Comment:
Just noticed this class is useless. To be removed from V3 and from all Drupal instances.